css/public_analytics.scss

```diff
@@ -66,19 +65,9 @@ $h3-font-size: rem(24px);
 $h4-font-size: rem(19px);
 $h5-font-size: rem(16px);
 
-$font-light: 300;
-
-h1,
-h2,
-h3,
-h4,
-h5 {
-  font-family: $font-sans;
-}
-
-.usa-display-alt {
-  font-size: rem(44px);
-  font-weight: $font-light;
+// @TODO: Refactor this with USWDS grid-container.
+.responsive-container {
```

Comment:
This used to be set explicitly in every element that used it. Setting it to a single class for compatibility.

Will need to refactor once USWDS is used with theme settings and/or when Neat library is removed.